tailwind.config.js

```diff
@@ -1,10 +1,13 @@
+const colors = require("tailwindcss/colors")
+
 module.exports = {
   mode: "jit",
   purge: ["./src/**/*.{ts,tsx}"],
```

Comment:
```suggestion
  content: ["./src/**/*.{ts,tsx}"],
```
Purge has been renamed to `content` (see [here](https://tailwindcss.com/docs/upgrade-guide#configure-content-sources))